staking/availability/measure_test.go

```diff
@@ -731,3 +731,60 @@ func makeTestWrapper(addr common.Address, numSigned, numToSign int64) staking.Va
 	val.Counters.NumBlocksSigned = new(big.Int).SetInt64(numSigned)
 	return val
 }
+
+type stateValidatorWrapperImpl struct {
+	v   *staking.ValidatorWrapper
+	err error
+}
+
+func (a stateValidatorWrapperImpl) ValidatorWrapper(addr common.Address, sendOriginal bool, copyDelegations bool) (*staking.ValidatorWrapper, error) {
+	return a.v, a.err
+}
+
+func TestUpdateMaxCommissionFee(t *testing.T) {
+	t.Run("0.6 + 0.6 = 1 (100%)", func(t *testing.T) {
```

Comment:
did you mean 0.6 + 0.4 ?

Reply:
no, it means we can't have more than 100%